Main.java

```diff
@@ -9,7 +11,23 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    // create scanner for user input
+    Scanner input = new Scanner(System.in);
+
+    //gets users first number
+    System.out.println("Please enter an integer:");
+    int a = input.nextInt();
+
+    //gets users second number
+    System.out.println("Please enter another integer:");
+    int b = input.nextInt();
```

Comment:
Good work. Generally a and b would be poor choices for variable names. Remember, variables should always be named using camelCase and describe what they are storing or being used for.